app/api/email_utils.py

```diff
@@ -74,3 +74,16 @@ def send_email_verification_message(user_name, email):
     html = render_template('email_confirmation.html', confirm_url=confirm_url, user_name=user_name)
     subject = "Mentorship System - Please confirm your email"
     send_email(email, subject, html)
+
+
+def send_email_about_accepted_request(user_name, email):
+    """Sends an html email message to the specified user.
+
+        Args:
```

Comment:
these 4 lines should be aligned/indented with the line 80 right? At least that's what I'm seeing throughout the code base